app/components/Navbar/menuLinks.js

```diff
@@ -15,7 +15,7 @@ export const menuLinks = {
   //  href: '/cover',
   //  title: 'Protect',
   //  description: '',
-  //},
+  // },
```

Comment:
How is this introduced if we enforce the eslint rules and have commit lint enabled? It has happened to me that I try to push a change and I cant because of these problems on files I did not worked on